packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -130,6 +134,7 @@ function computeRequestTracingInfo(request: RequestCompleteEvent) {
     _dd: {
       span_id: request.spanId!.toDecimalString(),
       trace_id: request.traceId!.toDecimalString(),
+      rule_psr: configuration && 'tracingSampleRate' in configuration ? configuration.tracingSampleRate : undefined,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ ‏`configuration` and `configuration.tracingSampleRate` should always be defined. Did you find some case were it's not, or is it just for extra prevention?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
      rule_psr: configuration.tracingSampleRate,
```

Reply:
This was a really interesting question. Yes `tracingSampleRate` is always defined in configuration. However `_dd` is not always defined in Resource events and thus added to some confusion on my end.

Reply:
Ah, gotcha. Yes, a lot of attributes are optional in RUM events, because we want to keep strong backward compatibility: some older SDK versions might not send a newly introduced attribute, so the backend/UI should handle both cases.